cmd/util_ref.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+)
+
+func fromStrings(namespace, name string) string {
```

Comment:
Name needs to be changed to reflect what this function does, as do the arg names as this seems to be used to combine a cluster:project

Reply:
It is also used for concatenating scope and answer key so I renamed it to `concatScope`